src/index.ts

```diff
@@ -7,19 +7,28 @@ export const mouseless = {
 
         // el might not be present for server-side rendering.
         if (el) {
-            MouseTrap.bind(binding.value, (e: ExtendedKeyboardEvent) => {
-                if (focus) {
-                    el.focus();
+            MouseTrap.bind(
+                binding.value instanceof Object ? binding.value.key : binding.value,
+                (e: ExtendedKeyboardEvent) => {
+                    if (focus) {
+                        el.focus();
+                    } else {
+                        el.click();
```

Comment:
@naveen168 I don't think we should leave the ``el.click()`` here because it might execute a function twice if it is already bound to the click event. Put an else in below ``if`` that calls the ``el.click();``?

Reply:
Done